docs/book/v3/quick-start.md

```diff
@@ -1,99 +1,177 @@
 # laminas-hydrator
 
-Hydration is the act of populating an object from a set of data.
+The laminas-hydrator component provides functionality for hydrating objects (which is the act of populating an object from a set of data) and extracting data from them.
 
-laminas-hydrator is a simple component to provide mechanisms both for hydrating
-objects, as well as extracting data sets from them.
+The component contains [concrete implementations](#available_implementations) for a number of common use cases, such as by using arrays, object methods, and reflection, and provides [interfaces](#base_interfaces) for creating custom implementations.
 
-The component consists of interfaces, and several implementations for common use cases.
+## Usage
 
-## Base Interfaces
+### Hydrating an Object
 
-### ExtractionInterface
+To hydrate an object with data, instantiate the hydrator and then pass to it the data for hydrating the object.
 
 ```php
-namespace Laminas\Hydrator;
+<?php
 
-interface ExtractionInterface
-{
-    /**
-     * Extract values from an object
-     *
-     * @return mixed[]
-     */
-    public function extract(object $object) : array;
-}
+use Laminas\Hydrator;
+
+$hydrator = new Hydrator\ArraySerializableHydrator();
+
+$data = [
+    'first_name' => 'James',
+    'last_name' => 'Kahn',
+    'email_address' => 'james.kahn@example.org',
+    'phone_number' => '+61 419 1234 5678',
+];
```

Comment:
```suggestion
$data = [
    'first_name'    => 'James',
    'last_name'     => 'Kahn',
    'email_address' => 'james.kahn@example.org',
    'phone_number'  => '+61 419 1234 5678',
];
```

See: https://docs.laminas.dev/laminas-coding-standard/v2/coding-style-guide/#2-7-arrays

Reply:
Thanks for pointing that out.

Reply:
This is going to take me quite some time to write. I might have to break this up in to several PRs